docs/docs/60-new-docs/50-user-guide/20-how-to-guides/05-installing-kargo-cli.md

```diff
@@ -50,4 +50,8 @@ To install:
 
 1. Regardless of your chosen download method, complete the installation by
    moving the binary to a location in your file system that is included in the
-   value of your `PATH` environment variable.
\ No newline at end of file
+   value of your `PATH` environment variable. For example,
+
+   ```shell
```

Comment:
Why is this example required?

Reply:
No reason as such but there is no harm adding an example here.

Reply:
I think this introduces the issue of people blindly copying the command example, instead of figuring out what is applicable to their local system.

Given this, I would not add this unless you have an explicit upside which outweighs this potential problem.

Reply:
I think this introduces the issue of people blindly copying the command example, instead of figuring out what is applicable to their local system.

Given this, I would not add this unless you have an explicit upside which outweighs this potential problem.